starter/script.js

```diff
@@ -1 +1,49 @@
 // put code here!
+confirm("I am ready to play!")
+var age = prompt("What's your age?");
+if (age is less than 13) {
+    console.log("you're underage, still want to play?");
+} else {
+    console.log("play it safe!");
+}
+console.log("You are stuck in the woods.'");
+console.log("Suddenly, you hear a growling noise'");
+var userAnswer = prompt("do you run or hide?");
+if (userAnswer = "yes") {
+    console.log("Run for your life!");
+} else {
+    console.log("go hide now!'");
+}
+console.log("oops, you dropped something onthe way");
+var userAnswer = prompt("do you go get it or no?");
+if (userAnswer = "yes") {
+    console.log("run and get your scarf");
+} else {
+      console.log("okay, but it's a bit chilly");
+}
+console.log("you still hear the noise, and take a peek");
+var loop = function() {
+  if(growlingBear === true) {
+    console.log("watch out!");
+  } else {
+    console.log("keep hiding");
+var growlingNoise = "bear";
+switch (growlingNoise) {
+  case "lion":
+  console.log("lions roaring");
+  break;
+}
+console.log("The noise is getting closer and closer");
+var userAnswer = prompt("can you spot the lions from your hiding spot?");
+if (userAnswer = "yes") {
+  console.log("run and get in the cave");
+} else {
+  console.log("begin tracing your path for rescue");
+}
+console.log("you see a clue");
+var userAnswer = prompt("do you pick pick it up or keep tracing?");
+if (userAnswer = "yes") {
+  console.log("congrats! you're almost there");
+} else {
+  console.log("you have a long way to go");
+}
```

Comment:
This a good start, but you'll need to nest coniditionals or break out the different choices into functions and then call those functions at the appropriate times to fulfill the criteria of the assignment. 

I think your switch will never log "lions roaring" because `growlingNoise` is always going to be "bear"

otherwise 👍 on camelCase variables and using `prompt()` and storing the result in a variable, indentation practices, and well-formed conditionals